webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -258,8 +258,11 @@ getCarouselTemplate: function() {
 return `<div class="mck-msg-box-rich-text-container km-card-message-container  km-div-slider">
             {{#payload}}
             <div class="km-carousel-card-template">
+            <div class="km-carousel-card-header-container">
+            {{#url}}<a href = {{url}} target="_blank"><span class="km-carousel-url-container"></span></a>{{/url}}
```

Comment:
@hardikparashar check with Suraj or Devashish if we will give "openLinkInNewTab" options here for URL click event as we do at other places. If, yes then need to make changes according to that.

Reply:
The openLinkInNewTab options is not received from the google assistant payload. I checked for that. Only the url is received.